shared/build.gradle.kts

```diff
@@ -62,6 +64,13 @@ kotlin {
 }
 
 android {
+    buildFeatures {
+        compose = true
+    }
+    composeOptions {
+        kotlinCompilerExtensionVersion = "1.5.1"
```

Comment:
Maybe move Compose compiler version to gradle.properties?
Or even reuse Compose Multilatform Compiler?

Reply:
> even reuse Compose Multilatform Compiler?

the versions usually don't match, we can't reuse them

Reply:
Possible reliable solutions:
1. add an internal mapping Kotlin -> androidx compiler, and apply `buildFeatures` + `composeOptions` implicitly in the Gradle plugin
2. exclude applying androidx compiler somehow, and always apply jetbrains compiler

Both require changes in our Gradle plugin

Reply:
> Ideally, Compose Multilatform Compiler should work for anroid too

I even say - must work, and it is how it works for Android now.